pkg/furl/paths_test.go

```diff
@@ -52,7 +53,7 @@ var _ = Describe("StartsWithHomeDirAsTilde", func() {
 var _ = Describe("ResolveTilde", func() {
 
 	It("resolves ~/ against current user's home directory", func() {
-		initialPath := "~/some/location"
+		initialPath := "~" + string(os.PathSeparator) + "some" + string(os.PathSeparator) +"location"
```

Comment:
it seems like a user should be able to send a value with either the os separator or forward slashes. Maybe this should be an additional test rather than a replacement.